setup.cfg

```diff
@@ -0,0 +1,38 @@
+[metadata]
+name=pytest-marker-bugzilla
+summary=py.test bugzilla integration plugin, using markers
+description-file=README.txt
```

Comment:
after #27 replace with README.rst, in order to have nice page on Python Index site.